Cargo.toml

```diff
@@ -31,6 +32,7 @@ version = "0.0.0"
 
 [workspace.dependencies]
 bitflags = "2.4.2"
+chrono = "0.4.34"
```

Comment:
I've limited the dependencies for chrono as much as I can before errors arise. I ended up needing `alloc`, `clock`, and `wasmbind`.

Reply:
I was debating if we should just remove this entirely in the Rust side to avoid this dependency, and have the the script that writes to disk add the timestamp to the files (which is just a line or two in the Node script).

Reply:
Good suggestion for moving the timestamping to the npm package Bill. I've gone ahead and made that change and dropped the chrono dependencies altogether.